database-mysql/dummyDataNew.js

```diff
@@ -339,57 +342,254 @@ function Data() {
 
 
 
+// var today = new Date();
+// var startMinutes = ((today.getHours() * 60) + today.getMinutes());
+// var startSeconds = today.getSeconds();
+
+
+// var count = 0;
+// var totalSize = 100000;
+// var batchSize = 10000;
+// var wrapper = () => {
+//   if(count >= totalSize) {
+//     console.log('count complete: ', count);
+//     return;
+//   }
+//   var promise = new Promise(
+//     function(resolve) {
+//     //setTimeout(() => {
+//       count += 1;
+//       ID = count;
+//       var data = new Data;
+//       db.interstAll(ID, data.name, data.roomType, data.roomTypeDetails, data.city, data.cityDetails, data.listingDetails, data.guestAccessDetails, data.interactionGuestsDetails, data.otherDetails, function (err, result) {
+//         if (err) {
+//           console.log(err);
+//         } else {
+//           // console.log('added: ', count)
+//           // const used = process.memoryUsage().heapUsed / 1024 / 1024;
+//           // console.log(`The script uses approximately ${Math.round(used * 100) / 100} MB`);
+//           // if (count === 100 || count === 1000 || count % 10000 === 0 || count === 100000 || count === 1000000) {
+//           //   const used = process.memoryUsage().heapUsed / 1024 / 1024;
+//           //   console.log(`The script uses approximately ${Math.round(used * 100) / 100} MB`);
+//           //   console.log('count: ', count);
+//           // }
+//           if (count === 100000) {
+//             console.log ('reached: ', count);
+//             var today = new Date();
+
+//             var minutes = ((today.getHours() * 60) + today.getMinutes()) - startMinutes;
+//             var seconds = today.getSeconds() - startSeconds;
+
+//             console.log('time elapsed:', minutes, 'm : ', seconds, 's');
+
+//           }
+//           resolve();
+//         }
+//       });
+
+//     //}, 0)
+//   });
+
+//   promise.then(function() {
+//     wrapper();
+//   });
+// }
+
+// wrapper();
+
+
+
+
+
+
+
+
+//===========================================
+var insertNumOfRows = (numOfRows, callback) => {
+  var all = [];
+  for (var i =0; i < numOfRows; i++) {
```

Comment:
minor but space after = in var i = 0, can also do formatDocument command+shift+p, and type in format document in the search bar